x-pack/plugins/canvas/public/components/expression/expression.scss

```diff
@@ -31,6 +31,15 @@
     flex-direction: column;
   }
 
+  .canvasExpressionInput__editor {
+    height: auto;
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 57px;
```

Comment:
You probably won't be surprised that I've asked, but how did you arrive at `57px`?

Reply:
It's the "natural" height of the bottom toolbar in the DOM.  I tried to find a way to calculate it with `$eui*` but I couldn't get it right where it didn't look insane: padding times 2, plus 1 for border, plus an unknown for the line height. :-/